### DISCUSSION TITLE
how to support ": after "pseudo-class and !important？

### DISCUSSION THREAD

**Discussion by yangjinjun3** [tailwindlabs/tailwindcss#2456] [Help]

**Comment by simonswiss** [comment]
Hey @yangjinjun3 👋

Would you care to add a bit more details to your question?

To support the `:after` pseudo-class, you could use the [`addVariant` function](https://tailwindcss.com/docs/plugins#adding-variants) available in the Tailwind CSS plugin API.

And you can find information about support `!important` with the [`important` config option](https://tailwindcss.com/docs/configuration#important).
